content/en/blog/install-ubuntu-utm-apple-silicon.md

```
@@ -180,28 +178,40 @@ The installation will copy and configure files, which may take some time. When p
 
 ::media
 ---
-source: https://cdn.xanzhu.com/v1/ubuntu-2504/installer-restart.webp
+source: https://cdn.xanzhu.com/2026/utm-ubuntu/os-restart.webp
 alt: Ubuntu prompting user to restart
 ---
 ::
 
-- **Apple Virtualization**: The VM restarts automatically.
-- **QEMU**: Eject the ISO manually by clicking the disc icon, selecting the ISO, and choosing **Eject**. Press **Enter** to reboot.
+#### Eject the ISO file
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)